RevenueCatUI/CustomerCenter/ViewModels/RelevantPurchasesListViewModel.swift

```diff
@@ -22,24 +22,33 @@ import SwiftUI
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
 @MainActor
-final class ActiveSubscriptionsListViewModel: BaseManageSubscriptionViewModel {
+final class RelevantPurchasesListViewModel: BaseManageSubscriptionViewModel {
 
     @Published
     private(set) var activePurchases: [PurchaseInformation] = []
 
+    @Published
+    private(set) var nonSubscriptionPurchases: [PurchaseInformation] = []
```

Comment:
Would it make sense to rename `activePurchases` to `activeSubscriptionPurchases`? 
Otherwise, it could be confusing as it might imply that it contain all `nonSubscriptionPurchases`.

Also, for consistency with naming in other files, should `nonSubscriptionPurchases` be called `activeNonSubscriptionPurchases`? Or perhaps it contains non-active nonsubscriptions?

Reply:
Make sense 👍